### DISCUSSION TITLE
Hi Pryorda, I‘d like to take over managing this project.

### DISCUSSION THREAD

**Discussion by akafusheng** [pryorda/vmware_exporter#396] [General]
I‘d like to take over managing this project. What should I do? Please let me know.

**Comment by erichester76** [comment]
I'd say fork it and start doing what you can

**Comment by mymaor89** [comment]
I would love to get a link to the forked project

**Comment by pryorda** [comment]
I would love to have someone take over this project. However, since its in many core infrastructure I will need proof of who you are. The work you've done in the past. Credible references to your work ethics.
